modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Button.js

```diff
@@ -29,7 +30,11 @@ export default function Button({
 	return (
 		<ClayButton className={cssClass} {...otherProps}>
```

Comment:
LGTM

Maybe after this one is merged we can update [`DropdownMenu`](https://github.com/liferay/liferay-portal/blob/3c6bbac29b64769b665e37798c7f51d4e45b6ff0/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js#L55) to use this component instead of `ClayButton`, so we don't duplicate this logic.

What do you think? 

cc @jbalsas ?

Reply:
Sounds good to me 👍